### PR TITLE
Add CallNamed() method to make call using named parameters.

### PR DESCRIPTION
**Pull request by rossmcf**
I've hastily added a method to allow method params to be sent by name, as described [here](http://www.jsonrpc.org/specification#parameter_structures).

I'd appreciate any feedback you could offer.

Thanks,
Ross

### CODE REVIEW

File: jsonrpc.go [hunk 1]
```diff
@@ -250,7 +276,7 @@ func (client *RPCClient) SetHTTPClient(httpClient *http.Client) {
 	client.httpClient = httpClient
 }
 
-func (client *RPCClient) newRequest(notification bool, method string, params ...interface{}) (*http.Request, error) {
```
Comment (ybbus) [inline jsonrpc.go]: Now you cannot do something like call("myMethod", "myParam1", 123, "myParam3") anymore, can you?

**Comment by ybbus** [comment]
Thanks for your PR.

Unfortunatly there was a problem with the test file. It seemed to be overwritten somehow.
So I reverted it.

Would be great if you could check your changes against the tests and add additional tests for the new code.

**Comment by rossmcf** [comment]
Hi @ybbus. I've fixed the issue shown by the tests, and added another test to cover `CallNamed()`. Hopefully that tidies it up.

**Comment by ybbus** [comment]
Hi Ross.

Thanks for your contribution.
Only thing I did was removing the time.Now from your tests, since this seems to be system dependent (UTC, GMT+X stuff).

Greetings,
Alex

**Comment by rossmcf** [comment]
👍 Thanks Alex.

**Pull request merged**